saleor/checkout/views/__init__.py

```diff
@@ -195,6 +195,25 @@ def update_cart_line(request, cart, variant_id):
     return JsonResponse(response, status=status)
 
 
+@get_or_empty_db_cart()
+def clear_cart(request, cart):
+    """Clear cart"""
+    from ..utils import update_cart_quantity
```

Comment:
It's safe to import it at the top of the file (accidental local import?)

Reply:
yeah, thats true. I always place import statement on top of file, but i saw this kind of import in another file so i think that this is proper way in this project. But good to know that I was wrong:)